hasher-matcher-actioner/hmalib/common/label_models.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import hmalib.common.config as config
+import json
+import typing as t
+
+from dataclasses import dataclass, field, fields
+from hmalib.common.logging import get_logger
+from requests import get, post, put, delete, Response
+
+logger = get_logger(__name__)
+
+
+@dataclass(unsafe_hash=True)
+class Label:
+    key: str
+    value: str
+
+    def to_dynamodb_dict(self) -> dict:
+        return {"K": self.key, "V": self.value}
+
+    @classmethod
+    def from_dynamodb_dict(cls, d: dict):
+        return cls(d["K"], d["V"])
+
+    def __eq__(self, another_label: object) -> bool:
+        if not isinstance(another_label, Label):
+            return NotImplemented
+        return self.key == another_label.key and self.value == another_label.value
```

Comment:
This should be the default implementation, what happened?

(You may need to do `dataclass(eq=False)` on children to prevent it from being overwritten)